packages/express-react/src/root-style-registry.tsx

```diff
@@ -1,7 +1,7 @@
 import { type PropsWithChildren, useState } from 'react'
 
 import {
-  createRootStyleCache,
```

Comment:
Here and below, replace client-side `createRootStyleCache` calls with `createStyleCache`. Note that although this is technically a breaking change, the symbol is public only through the`unstable-framework-support` API and therefore does not require a minor version bump.